### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by YigaUpdate**
I updated dependencies

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -20,10 +20,10 @@ env:
 #     CIRRUS_SHELL: "bash"
 #     COMPILER: msvc
 #     BUILD_ARTIFACTSTAGINGDIRECTORY: ${CIRRUS_WORKING_DIR}\artifacts\
-#     QT_VER_MSVC: 'msvc2019'
+#     QT_VER_MSVC: 'msvc2022'
 #     QT_DATE: '202310040911'
 #     QTDIR: C:\Qt\${QT_VER}\${QT_VER_MSVC}_64
-#     VULKAN_VER: '1.3.224.1'
```
Comment (elad335) [inline .cirrus.yml:26]: This is intentionally this specific version iirc. It is not tgat we forgot to update.

Comment (YigaUpdate) [reply .cirrus.yml:26]: OK I didn't know

Comment (Megamouse) [reply .cirrus.yml:26]: And neither is this change reflected in the submodules

**Comment by Megamouse** [comment]
I just realized this is all commented out lmao

**Comment by Megamouse** [comment]
Also, there's no Qt for msvc2022

**Pull request closed without merging**